<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditClientWidget</class>
 <widget class="QMainWindow" name="AuditClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuditClientWidget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>128</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion au serveur</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>759</width>
      <height>204</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Addresse IP du serveur :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEditPort"/>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEditAdresseIP"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Num√©ro de port du serveur :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Etat de la connexion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditEtat"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonQuitter">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Quitter</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonConnexion">
         <property name="text">
          <string>Connexion</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>611</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonOrdinateur">
           <property name="text">
            <string>nom de l'ordinateur distant</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonUtilisateur">
           <property name="text">
            <string>Nom de l'utilisateur</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonArchitecture">
           <property name="text">
            <string>Architecture processeur</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonOS">
           <property name="text">
            <string>OS de l'ordinateur</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEditOrdinateur"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditUtilisateur"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditArchitecture"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditOS"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>128</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Information Client</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
